### DISCUSSION TITLE
兆能m2如何开启无线支持？

### DISCUSSION THREAD

**Discussion by Canary233** [coolsnowwolf/lede#12016] [Q&A / 快速问答]
已经勾选了`ipq-wifi-zn_m2`和`ath11k-firmware-ipq6018`，仍然没有无线选项，内核日志也没看到ath11k驱动加载

**Comment by ysuolmai** [comment]
大雕说要修改dts, 不过我改了dts还是没有wifi, 不知道怎么解决
